criteria/active-community.md

```diff
@@ -0,0 +1,44 @@
+---
+order: 15
+---
+# Foster an active community
+
+## Requirements
+
+* Software MUST be in use by multiple parties
+* Communication channels MUST exist for users AND developers
+* The roadmap SHOULD be influenced by the needs of multiple parties
+* MAY hold regular events
+
+## Why this is important
+
+* Members of active communities support each other.
+* Codebases without communities are either too narrow for stewardship by the Foundation for Public code or must continue under incubation until a self-sustaining community can be built.
+
+## What this does not do
+
+* This does not shift responsibility for documentation, support, bug-fixing, or anything else to an unpaid community.
+
+## How to test
+
+* Count known users
+* Count messages per week
+* Count commits per week
+* Count distinct commit authors per week
+* Graph bug time-to-resolution over time
+* Count bugs and feature requests opened over time
+
+## Policy makers: what you need to do
+
+* Policy makers should ensure that their organization is listed in the known users.
+
+## Management: what you need to do
+
+* Ensure awareness of the community
```

Comment:
Suggestions:

Make sure devs, designers and policy people have time to contribute to the community
Prioritise community maintenance work